packages/alt-ergo/alt-ergo.0.95.2/opam

```diff
@@ -1,24 +1,34 @@
+(******************************************************************************
+* Please, do not modify this file without contacting the maintainers
+* to discuss the reasons of the modification.
+* In particular, dev-repo is not provided because it's private.
+******************************************************************************)
```

Comment:
I'm talking about this: https://github.com/OCamlPro-Iguernlala/opam-repository/commit/45d176a3f1b04594905ce518c4d8f85ad9a88996#diff-be2091939deb5d5787d4832b14dfa74fL15

Perhaps unnecessary with your update but I believe the `autoconf` dependency in the 0.95.2 release was necessary at least on Nov 17, 2013: https://github.com/ocaml/opam-repository/commit/8ad47a965c2fdedc72f53921be7d1b5f5a05e948